src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,6 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        return Integer.toBinaryString(value);
```

Comment:
please look at [checklist](https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_01.html). you should write you own realisation